README.md

```diff
@@ -1,55 +1,63 @@
-# Rails Sales Taxes Kata
-## Introduction
+# Rails Sales taxes
 
-You should provide sufficient evidence that your solution is complete by,
-as a minimum, indicating that it works correctly against the supplied test data.
+> Simple rails app where users upload a file with the content of a basket then prints the receipts on the web page.
 
-## PROBLEM
 
-Basic sales tax is applicable at a rate of 10% on all goods, except books, food, and medical products that are exempt.
-Import duty is an additional sales tax applicable on all imported goods at a rate of 5%, with no exemptions.
+## Built With
 
-When I purchase items, I receive a receipt that lists the name of all the items and their price (including tax),
-finishing with the total cost of the items and the total amounts of sales taxes paid.
+- Ruby v2.7.1
+- Ruby on Rails v6.0.4
 
-The rounding rules for sales tax are that for a tax rate of n%, a shelf price
-of p contains (np/100 rounded up to the nearest 0.05, ex. 0.5625 => 0.60, 1.649 => 1.65, 1.61 => 1.65)
-amount of sales tax.
+## Getting Started
 
-The application should only ask to the user to upload a file with the content of a basket and print the receipt on a
-web page.
+To get a local copy up and running follow these simple example steps.
 
-### INPUT
+### Prerequisites
 
-The files basket_1.txt, basket_2.txt, and basket_3.txt are the three input files for the application.
+Ruby: 2.7.1
 
-Write a Rails application that prints out the receipt details for the three baskets provided.
+npm: run <code> npm install </code>
 
-### EXPECTED OUTPUT
+yarn
```

Comment:
It's great that you wrote a README detailing the versions and dependencies 👏🏽. 

However, this plain line is confusing: are we supposed to install yarn? Run the yarn command? Think of the end-user and try to write instructions as simple and concrete as possible.